Newsletter2Go/Export/Model/PluginVersion.php

```diff
@@ -19,7 +19,7 @@ public function getStandardVersion()
         static $version;
 
         if (!$version) {
-            $composerPath = __DIR__ . '/../../composer.json';
+            $composerPath = __DIR__ . '/../composer.json';
```

Comment:
Attempted to read the file from the wrong path, this correctly fixes reading the version.